src/communication/azext_communication/version.py

```diff
@@ -4,7 +4,7 @@
 # --------------------------------------------------------------------------------------------
 
 
-VERSION = '1.8.0b1'
+VERSION = '1.8.1b1'
```

Comment:
Based on our [CLI Extension Version Transition Table](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md#cli-extension-version-transition-table), the version should be `1.8.0b2`. 
Please also update `azext.isPreview` to `true` in azext_metadata.json as well if you want to release as preview version.

Reply:
This has been updated per your other PR feedback